mmv1/products/memcache/api.yaml

```diff
@@ -198,3 +198,118 @@ objects:
             name: params
             description: |
               User-defined set of parameters to use in the memcache process.
+      - !ruby/object:Api::Type::NestedObject
+        name: maintenancePolicy
+        description: |
+          Maintenance policy for an instance.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'createTime'
+            output: true
+            description: |
+              Output only. The time when the policy was created.
+              A timestamp in RFC3339 UTC "Zulu" format, with nanosecond
+              resolution and up to nine fractional digits
+          - !ruby/object:Api::Type::String
+            name: 'updateTime'
+            output: true
+            description: |
+              Output only. The time when the policy was updated.
+              A timestamp in RFC3339 UTC "Zulu" format, with nanosecond
+              resolution and up to nine fractional digits.
+          - !ruby/object:Api::Type::String
+            name: 'description'
+            description: |
+              Optional. Description of what this policy is for.
+              Create/Update methods return INVALID_ARGUMENT if the
+              length is greater than 512.
```

Comment:
Do we want a validation on the string length for `description`?

Reply:
Good catch. I've added the validation (I hope in the right place). Thanks!